src/nupic/ntypes/Buffer.cpp

```diff
@@ -365,11 +367,13 @@ inline Int32 findWithLeadingWhitespace(const ReadBuffer &r, const char *s,
   return 0;
 }
 
-typedef NTA_Byte *(*fp_alloc)(NTA_UInt32);
-typedef void (*fp_dealloc)(NTA_Byte *);
+typedef Byte *(*fp_alloc)(UInt32);
```

Comment:
I'd call it some other name, it's kind of nasty overriding the expected meaning

Reply:
All I am doing is getting rid of the NTA_ prefix on some of the defined types.  These are redundant and not needed.  There are very few of those left in the code.